test_conformance/basic/test_sizeof.cpp

```diff
@@ -292,12 +292,18 @@ int test_sizeof(cl_device_id device, cl_context context, cl_command_queue queue,
             continue;
         }
 
-        if( gIsEmbedded &&
-           0 == strcmp(other_types[i], "image3d_t") &&
-           checkFor3DImageSupport( device ) == CL_IMAGE_FORMAT_NOT_SUPPORTED)
+        if( 0 == strcmp(other_types[i], "image3d_t" ) )
         {
-            log_info("\n3D images are not supported by this device. Skipping test.\t");
-            continue;
+            if( checkForImageSupport( device ) == CL_IMAGE_FORMAT_NOT_SUPPORTED)
+            {
+                log_info("\nimages are not supported by this device. Skipping test.\t");
+                continue;
+            }
+            if (gIsEmbedded && checkFor3DImageSupport(device) == CL_IMAGE_FORMAT_NOT_SUPPORTED)
```

Comment:
We could drop the check for `gIsEmbedded` here as `checkFor3DImageSupport` already handles the embedded case.

Reply:
Then we can also drop `checkForImageSupport`, as `checkFor3DImageSupport` also handles standard image support.